### PR TITLE
Adds `bdist_wininst` definition

### PR DESCRIPTION
**Pull request by sobolevn**
Source: https://github.com/python/cpython/blob/3.9/Lib/distutils/command/bdist_wininst.py

Removed in `3.10`

### CODE REVIEW

File: stdlib/@python2/distutils/config.pyi [hunk 1]
```diff
@@ -13,5 +13,3 @@ class PyPIRCCommand(Command):
     boolean_options: ClassVar[list[str]]
     def initialize_options(self) -> None: ...
     def finalize_options(self) -> None: ...
-    @abstractmethod
```
Comment (sobolevn) [inline stdlib/@python2/distutils/config.pyi:16]: I've also removed several redefinitions of `run` abstract method. It is defined in `Command`

File: stdlib/distutils/command/bdist_wininst.pyi [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import sys
+from _typeshed import StrOrBytesPath
+from typing import Any, ClassVar
+
+if sys.version_info < (3, 10):
```
Comment (srittau) [inline stdlib/distutils/command/bdist_wininst.pyi:5]: Instead of using this guard, the file should be added to `VERSIONS`.

Comment (sobolevn) [reply stdlib/distutils/command/bdist_wininst.pyi:5]: It is already there: https://github.com/python/typeshed/blob/5c309e5fe4a8d4898fe9f2b8664bb66cff845878/stdlib/VERSIONS#L116

**Comment by sobolevn** [comment]
Ok, looks like this commit finally passes: https://github.com/python/typeshed/pull/7249/commits/ebca8105aed047f09cd313c82aba5a2b1d213b1c

I pushed one more to test things without `if sys.version_info`: https://github.com/python/typeshed/pull/7249/commits/d2506c827d7dda8c9f137af36d8f2cadb8cdf21a

**Comment by sobolevn** [comment]
Yes, it failed 🎉

File: stdlib/distutils/command/bdist_wininst.pyi (hunk 2 above)
Comment (hauntsaninja) [reply stdlib/distutils/command/bdist_wininst.pyi:5]: I'm not sure I see the comments you're referring to, but cf https://github.com/python/mypy/pull/12083

Comment (sobolevn) [reply stdlib/distutils/command/bdist_wininst.pyi:5]: https://github.com/python/typeshed/pull/7249#issuecomment-1042909330

**Comment by JelleZijlstra** [comment]
We now run stubtest from mypy master with the VERSIONS fix, could you merge master and try removing the version check again?

**Comment by sobolevn** [comment]
Done!

**Pull request merged**